src/main/java/com/Amazon/Repository/CartRepository.java

```diff
@@ -101,7 +101,7 @@ public CommonResponse itemsCount(String userName) {
 			Session session = getSession();
             int result= (int)((long)session.createQuery("select count(p) from CartItem p where p.userName = :userName")
```

Comment:
why are we casting it to two different datatype here?

Reply:
will update